### DISCUSSION TITLE
Unable to save my project's Homepage URL in Domain Verification

### DISCUSSION THREAD

**Discussion by ECBSJ** [WalletConnect/walletconnect-monorepo#3554] [Developer Support] [answered]
Under Domain Verification, in my project's Cloud account, i'll input my project's homepage URL `https://happyhourdao.xyz` in the Homepage field but keeps on getting an error of `invalid input syntax for type uuid: "undefined"`

Please help!

**Comment by ArcQ** [comment]
same, we're trying to save https://parallel.life, it doesn't seem to be doing anything when you hit save

**Comment by glitch-txs** [comment, marked as answer]
Hi, it should be fixed now, please feel free to ping me if this issue persists. Thanks!
